training/INFRASTRUCTURE.md

```diff
@@ -0,0 +1,29 @@
+Cloud Services and Infrastructure in Digital Publishing
+===========================
+
+We use Amazon Web Services in Digital Publishing to host all of our services, as well as our Continuous Integration (CI).
+
+You should be given relevant AWS accounts by your manager or a Tech Lead - speak to your manager if this does not happen soon upon starting with the team. 
+
+
+Introduction to our Infrastructure
+----------------------------
+
+In your first few weeks you should be invited to a number of 'Intro Workshops', one of which will be on the topic of Infrastructure. 
+
+### Pre reading
+
+- Make sure you're familiar with basic AWS terms: 
+    -  [Availability Zones and Regions](https://aws.amazon.com/about-aws/global-infrastructure/regions_az/) 
+    - VPCs and Subnets are also good to understand at a basic level
+
+### Materials
+
+The workshop will walk through [these slides (WIP)](https://docs.google.com/presentation/d/1VQjYd6R6xDRluA_fTF4teu97umJam2y--XlewZjeBrU/edit?usp=sharing) over roughly 1 hour, with time for questions.
+
+
+
+Advanced Infrastructure
```

Comment:
In the interest of creating a standard template of the sections of these pages perhaps this could be:
```suggestion
Further Reading
```
Or
```suggestion
Further Resources
```